packages/react-components/src/badge/src/Badge.css

```diff
@@ -96,6 +110,15 @@
     opacity: 0.4;
 }
 
+/* ICON */
+.o-ui.badge.icon {
+    display: inline-flex;
```

Comment:
Why would you need to alter an icon display to flex? It can't have any content.

Reply:
We needed it when I tried to go overboard with a background and all. Not needed anymore.